README.md

```diff
@@ -86,6 +86,14 @@ pip install tox
 tox
 ```
 
+To build documentation when you changed something in `docs`:
+
+```sh
+pip install sphinx
+sphinx-build docs build/html
+xdg-open build/html/index.html
```

Comment:
I'm not familiar with xdg-open, can we just omit this line? I think it's safe to assume folks will know how to or can figure out how to view the generated HTML